types/transaction_fork_hash.go

```diff
@@ -47,7 +47,9 @@ func (th *TransactionHashForkV1) ComputeHash(t *Transaction) {
 	t.ChainID = big.NewInt(0)
 
 	v := t.MarshalRLPWith(ar)
-	hash.WriteRlp(t.Hash[:0], v)
+	hashTmp := ZeroHash
```

Comment:
can we have here `var hashTmp types.Hash`. We don't need unnecessary copy

Reply:
Isn't it the same in the end? `ZeroHash` is a global variable and I don't understand which copy are you referring to?

Reply:
Yeah, I have checked with chatgpt it seams that both approaches has same performance. I thought that hashTmp := ZeroHash is less performant